zerver/views/auth.py

```diff
@@ -855,6 +855,61 @@ def view(request, *args, **kwargs):
     return two_fa_view(request, **kwargs)
 
 
+@csrf_exempt
+@has_request_variables
+def jwt_fetch_api_key(request: HttpRequest, json_web_token: str = REQ(default="")) -> HttpResponse:
```

Comment:
Most of the logic here seems to just be taken from `remote_user_jwt`. Could we just extract it into a helper function that'll be called in both functions, to avoid the duplication?

Reply:
The helper function has been added to the PR, closing this conversation since we're discussing in the other one.